pkg/exporter/syslogexporter/README.md

```diff
@@ -14,21 +14,25 @@ The syslog exporter supports sending messages to a remote syslog server.
   This ensures that all the syslog message headers are populated with the expected values.
 - Not using the `syslog_parser` will result in the syslog message being populated with default header values.
 
-__Note__: Syslog over UDP doesn't support certificate verification (`ca_certificate`).
-
 ## Configuration
 
 **The following are a few configuration options available to forward syslog messages**:
 
-- `endpoint` - syslog endpoint (FQDN or IP address)
-- `protocol` - tcp/udp
-- `port` - A syslog port
-- `format` - rfc5424/rfc3164
+- `endpoint` - (default = `host.domain.com`) syslog endpoint (FQDN or IP address)
+- `protocol` - (default = `tcp`) tcp/udp
+- `port` - (default = `514`) A syslog port
+- `format` - (default = `rfc5424`) rfc5424/rfc3164
   - `rfc5424` - Expects the syslog messages to be rfc5424 compliant
   - `rfc3164` - Expects the syslog messages to be rfc3164 compliant
-- `ca_certificate` [tcp only] - A publicly verifiable server certificate (`note`: Self signed certificates are not supported in this version)
-- `certificate` [tcp only] - certificate for mTLS communication (client certificate)
-- `key` [tcp only] -  Key for mTLS communication (client key)
+- `tls` - configuration for TLS/mTLS
+  - `insecure` (default = `false`) whether to enable client transport security, by default, TLS is enabled.
+  - `cert_file` - Path to the TLS cert to use for TLS required connections. Should only be used if `insecure` is set to `false`.
+  - `key_file` - Path to the TLS key to use for TLS required connections. Should only be used if `insecure` is set to `false`.
+  - `ca_file` - Path to the CA cert. For a client this verifies the server certificate. For a server this verifies client certificates. If empty uses system root CA. Should only be used if `insecure` is set to `false`.
+  - `insecure_skip_verify` -  (default = `false`) whether to skip verifying the certificate or not.
```

Comment:
are these options built into the `configtls`?

Reply:
please see here: https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/configtls.go, these options are used to created tls config